product.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <vector>
+
+int Answer(std::vector<int> v){
```

Comment:
- Add a function comment: what does it do, what are the parameters/return values
- Make sure to follow our style guide for functions NamedLikeThis
- Put the function implementation in FunMath.cpp and the function definition in FunMath.h
- Give this function a more descriptive name.